applications/application-health.adoc

```diff
@@ -29,5 +29,5 @@ include::modules/odc-monitoring-health-checks.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* For details on switching to the *Developer* perspective in the web console, see xref:../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[About *Developer* perspective].
+* For details on switching to the *Developer* perspective in the web console, see xref:../web_console/web-console-overview.adoc#about-developer-perspective_web-console-overview[About *Developer* perspective].
```

Comment:
Consider "About the *Developer* perspective". Per the IBM Style Guide, articles shouldn't be omitted.